source/material_model/grain_size.cc

```diff
@@ -1232,6 +1232,12 @@ namespace aspect
                              Patterns::Bool (),
                              "This parameter determines whether to use bilinear interpolation "
                              "to compute material properties (slower but more accurate).");
+          prm.declare_entry ("Phase distribution parameter", "0.72",
+                             Patterns::Double (0.),
+                             "This parameter is equal to $3*\\phi_1*\\phi_2*, where $\\phi_1$ "
```

Comment:
Depending on what we find about the number 3, we would only want to read in phi_1 * phi_2 here, if 3 is the geometric constant. We could even think about reading them in separately? Let's discuss this today.